.drone.star

```diff
@@ -0,0 +1,100 @@
+OC_CI_PHP = "owncloudci/php:%s"
+OC_UBUNTU = "owncloud/ubuntu:20.04"
+PLUGINS_GITHUB_RELEASE = "plugins/github-release"
+
+DEFAULT_PHP_VERSION = "8.1"
+
+dir = {
+    "base": "/var/www/owncloud",
+}
+
+config = {
+    "branches": [
+        "main",
+    ],
+    "codestyle": True,
+    "phpstan": True,
+    "phan": {
+        "multipleVersions": {
+            "phpVersions": [
+                DEFAULT_PHP_VERSION,
+                "8.2",
+            ],
+        },
+    },
+    "phpunit": True,
+}
+
+
+def main(ctx):
+    return tests(
+        ctx,
+        [
+            ["codestyle", "make test-php-style"],
+            ["phpstan", "make test-php-phpstan"],
+            ["phan", "make test-php-phan"],
+        ],
+    ) + phpunit(ctx, [8.1, 8.2])
+
+
+def tests(ctx, tests):
+    pipelines = []
+    for index, test in enumerate(tests):
+        if test[0] not in config:
+            return pipelines
+        pipelines += [
+            {
+                "kind": "pipeline",
+                "name": test[0],
+                "steps": [
+                    {
+                        "name": str(index)+test[0],
```

Comment:
```suggestion
                        "name": test[0],
```

I don't think that we need the index. For example, https://drone.owncloud.com/owncloud/ocis-php-sdk/30/1/2 step is called "0codestyle". That looks strange.